airflow/providers/amazon/aws/hooks/batch_waiters.py

```diff
@@ -120,7 +120,7 @@ def default_config(self) -> Dict:
         :rtype: Dict
         """
         if self._default_config is None:
-            config_path = Path(__file__).with_name("batch_waiters.json").absolute()
+            config_path = Path(__file__).with_name("batch_waiters.json").resolve()
             with open(config_path) as config_file:
                 self._default_config = json.load(config_file)
```

Comment:
We don’t even need to resolve here, this should just work:

```python
with Path(__file__).with_name("batch_waiters.json").open() as config_file:
    self._default_config = json.load(config_file)
```

Reply:
We do not need to, but I prefer to do it in all cases where I am running any kind of path calculation that I am going to use later. The reason is that if I decide at a later point to use the path further and run it trough any command line tool, I do not have to worry whether it is absolute or not.